.github/workflows/aws-deploy.yml

```diff
@@ -28,11 +40,8 @@ jobs:
           actions_comment: false
           cdk_args: '--context env=dev --debug'
 
-      - name: Install dependencies
-        run: pip install -r requirements.txt -r requirements-dev.txt
-
       - name: Run unit tests
-        run: python -m pytest tests/ -s -v
+        run: venv/bin/python -m pytest tests/ -s -v
```

Comment:
This is not really "unit" tests anymore. It's everything (Unit and integration tests).

Also - Do we get value out of these running? Some thoughts:

1. In order to build the docker container in the first place **we would have needed to get a successful build**. Perhaps we should really **only run unit tests here as a cursory check** to verify that dependencies are working correctly, but not that the integration tests are still functioning.
2. With the changes we're working on in https://github.com/Sage-Bionetworks/schematic/pull/1512 - The intention is that we will be able to run a number of integration tests against the dev version of schematic deployed to AWS. Perhaps this is enough for us to consider only running the unit test directory here `tests/unit/`

Reply:
I glossed over the fact that it's this directory: https://github.com/Sage-Bionetworks/schematic-infra/tree/dev/tests that is being run - NOT, the testing directory of the schematic API/python code